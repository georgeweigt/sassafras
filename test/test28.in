data ;
input x @@ ;
datalines ;
1 2 3 4 5 6 7 8 9 10
;
proc means n mean clm maxdec=6 ;
run ;
