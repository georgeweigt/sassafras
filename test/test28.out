          Variable      N         Mean     95% CLM MIN     95% CLM MAX
          X            10     5.500000        3.334149        7.665851

